airflow/docker_image.go

```diff
@@ -14,6 +14,8 @@ import (
 	"regexp"
 	"strings"
 
+	"github.com/astronomer/astro-cli/airflow/runtimes"
+
```

Comment:
Just bumped all this new container runtime management stuff into a sub package `runtimes`, so that's just getting updated below.